src/handler.py

```diff
@@ -58,7 +58,7 @@ def _get_log_type(log_type=None):
     """
     if log_type:
         return log_type
-    return os.getenv("LOG_TYPE", "")
+    return os.getenv("LOG_TYPE") or os.getenv("LOGTYPE", "")
```

Comment:
I guess we can then write this as just (I'm not sure, my Python skills are a little rusty at the moment):

```
return log_type or os.getenv("LOG_TYPE") or os.getenv("LOGTYPE", "")
```

Reply:
Python may interpret "" as false, meaning that if the parameter is an empty string, that would pass. Anyway, these simple tests should reveal if this is the case. If so, let's simplify the code, like you suggest.

Reply:
Good point. I have no idea either. 😓